sdk/src/main/java/com/silanis/esl/sdk/SigningLogo.java

```diff
@@ -0,0 +1,25 @@
+package com.silanis.esl.sdk;
+
+import java.io.Serializable;
+import java.util.Locale;
+
+public class SigningLogo implements Serializable {
+    private Locale language;
+    private String image;
+
+    public Locale getLanguage() {
+        return language;
+    }
+
+    public void setLanguage(Locale language) {
+        this.language = language;
+    }
+
+    public String getImage() {
+        return image;
+    }
+
+    public void setImage(String image) {
+        this.image = image;
+    }
+}
```

Comment:
This part has already been modified and doesn't need to be modified. Please rebase the dev branch.